actions.go

```diff
@@ -88,11 +88,15 @@ func (c *Action) IssueCommand(cmd *Command) {
 // with the 'Command' argument as it's scope is unclear in the current
 // TypeScript implementation.
 func (c *Action) IssueFileCommand(cmd *Command) error {
+	return c.issueFileCommand(cmd, os.Getenv)
+}
+
+func (c *Action) issueFileCommand(cmd *Command, f getenvFunc) error {
```

Comment:
I have created unexported functions in-place of suggested `IssueFileCommandLookup`. _IMO_, exporting these functions doesn't create any value. I've also found this pattern to be a common occurrence in Kubernetes code-base. Please let me know if you think that these should be exported functions instead.

Reply:
Unexported is fine. Less to worry about maintaining/breaking in the future 😄